sdk/securityinsights/Azure.ResourceManager.SecurityInsights/src/Generated/OperationalInsightsWorkspaceSecurityInsightsResource.cs

```diff
@@ -2231,7 +2231,7 @@ public virtual Response<SecurityInsightsDataConnectorResource> GetSecurityInsigh
         /// </item>
         /// <item>
         /// <term>Operation Id</term>
-        /// <description>ListGeodataByIP</description>
+        /// <description>ListGeodataByIp</description>
```

Comment:
it's `Ip`: https://github.com/Azure/azure-rest-api-specs/blob/49456eb149e3bb5d0528cbca714cfa1806415484/specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2024-01-01-preview/EnrichmentWithWorkspace.json#L48